cogs/Animals.py

```diff
@@ -0,0 +1,36 @@
+import discord

```

Comment:
this is pretty minor, but the convention I prefer to follow is to name modules with lowercase names (ie `animals.py`, not `Animals.py`).

Reply:
Renamed all of them